#goal definition
float32 x_pickup
float32 y_pickup
float32 z_pickup
---
#result definition
float32 weight
---
#feedback
float32 x_current
float32 y_current
float32 z_current